src/Console/CommandEvent.php

```diff
@@ -24,10 +24,10 @@ class CommandEvent
      * @param $command
      * @param $input
      * @param $output
-     * @param $exception
+     * @param \Exception|\Error $exception
```

Comment:
Let's use \Throwable.

Reply:
\Throwable is php7 only, but I am fine using it here anyway.